src/ts/components/subtitlesettings/fontopacityselectbox.ts

```diff
@@ -1,37 +1,41 @@
-import {SubtitleSettingSelectBoxConfig, SubtitleSettingSelectBox} from './subtitlesettingselectbox';
+import {SubtitleSettingSelectBox} from './subtitlesettingselectbox';
 import {UIInstanceManager} from '../../uimanager';
-import {StorageUtils} from '../../storageutils';
-import {ColorUtils} from '../../colorutils';
 
 /**
  * A select box providing a selection of different font colors.
  */
 export class FontOpacitySelectBox extends SubtitleSettingSelectBox {
 
-  constructor(config: SubtitleSettingSelectBoxConfig) {
-    super(config);
-  }
-
   configure(player: bitmovin.PlayerAPI, uimanager: UIInstanceManager): void {
     super.configure(player, uimanager);
 
-    this.addItem('0.25', '25%');
-    this.addItem('0.50', '50%');
-    this.addItem('0.75', '75%');
-    this.addItem('1', '100%');
+    this.addItem(null, 'default');
+    this.addItem('100', '100%');
+    this.addItem('75', '75%');
+    this.addItem('50', '50%');
+    this.addItem('25', '25%');
 
-    this.selectItem('1');
+    this.onItemSelected.subscribe((sender, key: string) => {
+      this.settingsManager.fontOpacity.value = key;
 
-    if (StorageUtils.hasLocalStorage()) {
-      let color = window.localStorage.getItem('fontColor');
-      if (color != null) {
-        let col = ColorUtils.colorFromCss(color, ColorUtils.foreground);
-        this.selectItem(col.a.toString());
+      // Color and opacity go together, so we need to...
```

Comment:
For the background and window, with a default of 0%,it works ok.

For the font where the default opacity is 100%, setting the opacity to default should not reset the color I think.